katas/content/qubit/index.md

```diff
@@ -134,13 +134,13 @@ We will learn more about Dirac notation in the next katas, as we introduce quant
     "title": "Relative and Global Phase"
 })
 
-Complex numbers have a parameter called the phase. If a complex number $z = x + iy$ is written in polar form $z = re^{i\theta}$, its phase is $\theta$, where $\theta = \atan2(y, x)$.
+Complex numbers have a parameter called the phase. If a complex number $z = x + iy$ is written in polar form $z = re^{i\theta}$, its phase is $\theta$, where $\theta = atan2(y, x)$.
 
 > `atan2` is a useful function available in most programming languages. It takes two arguments and returns an angle $\theta$
 > between $-\pi$ and $\pi$ that has $\cos \theta = x$ and $\sin \theta = y$. Unlike using $\tan^{-1}(\frac{y}{x})$, `atan2` computes
 > the correct quadrant for the angle, since it preserves information about the signs of both sine and cosine of the angle.
 
-The probability amplitudes $\alpha$ and $\beta$ are complex numbers, therefore $\alpha$ and $\beta$ have a phase. For example, consider a qubit in state $\frac{1 + i}{2}|0\rangle + \frac{1 - i}{2}|1\rangle$. If you do the math, you see that the phase of $|0\rangle$ is $\atan2(\frac12, \frac12) = \frac{\pi}{4}$, and the phase of $|1\rangle$ is $\atan2(\frac12, -\frac12) = -\frac{\pi}{4}$. The difference between these two phases is known as **relative phase**.
+The probability amplitudes $\alpha$ and $\beta$ are complex numbers, therefore $\alpha$ and $\beta$ have a phase. For example, consider a qubit in state $\frac{1 + i}{2}|0\rangle + \frac{1 - i}{2}|1\rangle$. If you do the math, you see that the phase of $|0\rangle$ is $atan2(\frac12, \frac12) = \frac{\pi}{4}$, and the phase of $|1\rangle$ is $atan2(\frac12, -\frac12) = -\frac{\pi}{4}$. The difference between these two phases is known as **relative phase**.
```

Comment:
\atan2 here is deliberate, to use the same spelling as trigonometric functions

Reply:
But \atan2 isn't a valid latex command (in MathJax or Katex) and gives errors when used. (\arctan is, but that's not what you want).

Reply:
Right now on QCOM \atan2 renders successfully:
![image](https://github.com/microsoft/qsharp/assets/10113024/ec5760b7-84e3-40c6-a685-11270daafb9f)

Another discrepancy that we need to track down and fix on QCOM?

Reply:
I think QCOM just has MathJax set to 'ignore all errors' mode. If you go to the MathJax demo page at https://www.mathjax.org/#demo and put it in you'll see it's highlighted as an error.